components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -0,0 +1,292 @@
+| `operation_params` | Additional operation parameters. Possible values: `demo`, `poc`, `poc15Days`, `poc15DaysGuideMode`, `poc30Days`, `poc6months`, `production` – for updating the project type, `payAsYouGo` – for logging into a PAYG project, `googleLogin` – for logging into the project via Google SSO, `password` – for password-related operations (including login via credentials) | `password` |
```

Comment:
payg -> pay as you go a nalinkovat https://help.keboola.com/management/payg-project/

cely operation_params bych asi vytahl do zvlastni tabulky